frontend/src/components/CigCanvas.vue

```diff
@@ -84,6 +103,7 @@ export default defineComponent({
     image: {type: File, default: null},
     sendCanvas: {type: Boolean, default: false},
     savePictureTrigger: {type: Boolean, default: false},
+    index: {type: Number, default: null},
   },
   setup(props: CigCanvasProps, { root, emit }: SetupContext) {
     let name: Ref<string> = ref('');
```

Comment:
Type definition seems redundant.
```suggestion
    let name = ref('');
```